config.go

```diff
@@ -54,17 +54,17 @@ const (
 	defaultAllowHighFees       = false
 
 	// ticket buyer options
-	defaultMaxFee            dcrutil.Amount = 1e8
+	defaultMaxFee            dcrutil.Amount = 1e7
 	defaultMinFee            dcrutil.Amount = 1e6
 	defaultMaxPriceScale                    = 2.0
 	defaultMinPriceScale                    = 0.7
 	defaultAvgVWAPPriceDelta                = 2880
 	defaultMaxPerBlock                      = 5
 	defaultBlocksToAvg                      = 11
-	defaultFeeTargetScaling                 = 1.05
+	defaultFeeTargetScaling                 = 1.0
```

Comment:
I'm okay with reducing this, but I would prefer if we made the default 1.01 instead of 1.0, since 1.0 would essentially not scale at all and tickets could have a higher likelihood of being left unmined